spec.md

```diff
@@ -1616,8 +1616,10 @@ module MyModule :
 ```
 
 This is an illegal FIRRTL circuit and an error will be thrown during compilation.
-All wires, memory ports, instance ports, and module ports that can be connected to must be connected to under all conditions.
+All non-zero-width wires, memory ports, instance ports, and module ports that can be connected to must be connected to under all conditions.
```

Comment:
This and the next are ambiguous as it's not clear (at least looking at each sentence in isolation) whether "non-zero-width" only applies to wires or to the entire list.

Reply:
i agree that there seems to be a difference in use case for 0-width elements (SInt, UInt, Analog) and 0-element Aggregates (Vec, bundle types). It seems like users should be responsible for the first one, while the latter does make sense to allow to not happen in the spec. I'm actually not sure what the proposed spec change is currently trying to cover --both of these cases?